modules/ROOT/edits/enable-inbound-traffic-self.adoc

```diff
@@ -0,0 +1,22 @@
+= Configure Domains and Inbound Traffic (On Self-Managed Kuberenetes)
+
+To enable external applications to connect to Runtime Fabric on Self-Managed Kubernetes, you must provide an ingress controller that is compatible with your Kubernetes environment. Runtime Fabric uses the ingress controller to route requests to deployed applications.
+
+Inbound traffic is enabled by default. This allows Mule applications and API gateways to receive inbound connections. You can disable inbound traffic per application in Runtime Manager when deploying or configuring an application.
```

Comment:
Do we need to comment about why/under what circumstances our audience would want or need to disable inbound traffic?